pkg/apis/addtoscheme_olm.go

```diff
@@ -18,12 +18,10 @@ limitations under the License.
 package apis
 
 import (
-	olmv1 "github.com/operator-framework/operator-lifecycle-manager/pkg/api/apis/operators/v1"
-	olmv1alpha1 "github.com/operator-framework/operator-lifecycle-manager/pkg/api/apis/operators/v1alpha1"
+	scheme "github.com/operator-framework/operator-lifecycle-manager/pkg/api/client/clientset/versioned/scheme"
```

Comment:
I'd suggest to depend on `github.com/operator-framework/api` instead of `github.com/operator-framework/operator-lifecycle-manager/pkg/api` so that we can remove the dependency on the later, e.g.:

```suggestion
import (
	operatorsv1 "github.com/operator-framework/api/pkg/operators/v1"
	operatorsv1alpha1 "github.com/operator-framework/api/pkg/operators/v1alpha1"
	operatorsv1alpha2 "github.com/operator-framework/api/pkg/operators/v1alpha2"
)

func init() {
	// Register the types with the Scheme so the components can map objects to GroupVersionKinds and back
	AddToSchemes = append(AddToSchemes, operatorsv1.AddToScheme, operatorsv1alpha1.AddToScheme, operatorsv1alpha2.AddToScheme)
}
```

Reply:
Thanks!  I made the suggested changes.